### PR TITLE
psexec: add page

### PR DESCRIPTION
**Pull request by CAG07**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
@CAG07 Welcome to tldr, This looks more like a man page than a cheat sheet. Please take a look at our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) and update this page.

> Thanks in advance for your contribution.

**Comment by CAG07** [comment]
Thank you @kbdharun . I removed all of the optional parameters, but left the possible options under Usage. I can remove those too if needed.

**Review by kbdharun — COMMENTED** [review]
I think this would be better

```md
# PsExec

> Execute a command-line process on a remote machine.
> This is an advanced command and it might potentially be dangerous.
> More information: <https://learn.microsoft.com/en-us/sysinternals/downloads/psexec>.

- Execute a command using cmd in a remote shell:

`psexec {{ip_address | computer_name}} cmd`

- Execute a command on a remote computer:

`psexec {{ip_address | computer_name}} -u {{user_name}} -p {{password}}

- Execute a command remotely and output the result to a file:

`psexec {{ip_address | computer_name}} cmd /c {{command}} -an ^>{{path/to/file.txt}}

- Execute a program to interact with users:

`psexec {{ip_address | computer_name}} -d -i {{program_name}}
```

I think giving real-world examples for this command would be better than explaining the flags. Can you edit your page to this?

Reference: [1](https://medium.com/hacker-toolbelt/my-sysinternals-pstools-cheat-sheet-781fdf2e5e3a), [2](https://www.lifewire.com/psexec-4587631#toc-psexec-command-examples), [3](https://learn.microsoft.com/en-us/sysinternals/downloads/psexec).

@CleanMachine1 @navarroaxel @mfrw What do you think about this change?

**Comment by suyashbhawsar** [comment]
> {{ip_address | computer_name}}

What is `computer_name` here?

**Comment by kbdharun** [comment]
> > {{ip_address | computer_name}}
> 
> What is `computer_name` here?
> 
> 

I think `host_name` would be a better fit here. Thanks will update it.

**Comment by suyashbhawsar** [comment]
> > > {{ip_address | computer_name}}
> > 
> > 
> > What is `computer_name` here?
> 
> I think `host_name` would be a better fit here. Thanks will update it.

I'd recommend to directly use `hostname` as it's a common word in other commands.

**Comment by CleanMachine1** [comment]
Why not use what the ssh page uses
`remote_host`

**Comment by kbdharun** [comment]
> Why not use what the ssh page uses `remote_host`

Yeah, This is better I will update it.

**Comment by CAG07** [comment]
Updated to `remote_host`
Added \\ because that is required syntax 
Added a safe command ipconfig /all in the example

File: pages/windows/psexec.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Execute a command-line process on a remote machine.
+> This is an advanced command and it might potentially be dangerous.
+> More information: <https://learn.microsoft.com/en-us/sysinternals/downloads/psexec>.
+
+- Execute a command using cmd in a remote shell:
+
+`psexec \\{{remote_host}} cmd`
+
+- Execute a command on a remote computer:
```
Comment (kbdharun) [inline pages/windows/psexec.md:11]:
```suggestion
- Execute a command on a remote host (pre-authenticated):
```

File: pages/windows/psexec.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# PsExec
+
+> Execute a command-line process on a remote machine.
+> This is an advanced command and it might potentially be dangerous.
+> More information: <https://learn.microsoft.com/en-us/sysinternals/downloads/psexec>.
+
+- Execute a command using cmd in a remote shell:
```
Comment (kbdharun) [inline pages/windows/psexec.md:7]:
```suggestion
- Execute a command using `cmd` in a remote shell:
```

File: pages/windows/psexec.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Execute a command remotely and output the result to a file:
+
+`psexec \\{{remote_host}} cmd /c {{command}} -an ^>{{path/to/file.txt}}`
+
+- Execute a program to interact with users:
+
+`psexec \\{{remote_host}} -d -i {{program_name}}`
+
+- Example:
```
Comment (kbdharun) [inline pages/windows/psexec.md:23]:
```suggestion
- Display the IP configuration of the remote host:
```
or 

```suggestion
- Display `ipconfig` of remote host:
```

File: pages/windows/psexec.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# PsExec
```
Comment (CleanMachine1) [inline pages/windows/psexec.md:1]:
```suggestion
# psexec
```

Comment (kbdharun) [reply pages/windows/psexec.md:1]: > ```suggestion
> # psexec
> ```

I have a doubt recently I came to know there wouldn't be any linter errors for using Capital characters in heading. I even check this file in `tldrl`. Is this the cause of error here?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Yay, 🎉 the errors are fixed now.

**Review by blueskyson — APPROVED** [review]
Look good to me.

**Pull request merged**